src/OrleansTestingHost/TestingSiloHost.cs

```diff
@@ -661,13 +677,29 @@ public static SiloHandle StartOrleansSilo(TestingSiloHost host, Silo.SiloType ty
 
             // Load initial config settings, then apply some overrides below.
             ClusterConfiguration config = new ClusterConfiguration();
-            if (options.SiloConfigFile == null)
+            try
             {
-                config.StandardLoad();
+                if (options.SiloConfigFile == null)
+                {
+                    config.StandardLoad();
```

Comment:
My guess is that it is more likely that we break somebody by removing the config files from the nuget (which I think we should do) than by switching to programmatic config.

If somebody indeed edited the default config file, it's very easy to specify it in `clientOptions.ClientConfigFile` to get everything working as is. We'll need to highlight this change and explain the available options in the release notes.